cypress/support/pages/tasks/pageTasks.js

```diff
@@ -100,27 +114,20 @@ class Tasks {
   }
 
   setTaskText (taskText) {
-    cy.get(Tasks.#homeworkDescription)
-      .find('div > p')
-      .clear()
-    cy.get(Tasks.#homeworkDescription)
-      .find('div > p')
-      .type(taskText)
+    cy.get(Tasks.#homeworkDescription).find('div > p').clear()
```

Comment:
Can you name this selector for better handling later on `div > p`? It would be great

Reply:
I will have a look on it